lib/kafka_ex/protocol/offset_commit.ex

```diff
@@ -1,11 +1,11 @@
 defmodule KafkaEx.Protocol.OffsetCommit do
   defmodule Request do
-    defstruct consumer_group: "kafka_ex", topic: "", partition: 0, offset: 0, metadata: ""
+    defstruct consumer_group: "kafka_ex", topic: nil, partition: nil, offset: nil, metadata: ""
```

Comment:
I agree.  I wanted to do that in a separate PR because it has some other implications.